src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+
+public class HelloWorld {
+
```

Comment:
why do you remove all variables from this class?

Reply:
Reslove in new comit